lib/__tests__/standalone-fix.test.js

```diff
@@ -296,24 +298,25 @@ it('one rule being disabled and another still autofixing', async () => {
 		code,
 		config: {
 			rules: {
-				indentation: [
-					2,
+				indentation: [0],
+				'property-case': [
+					'lower',
 					{
 						disableFix: true,
 					},
 				],
-				'property-case': ['lower'],
 			},
 		},
 		fix: true,
 	});
 
-	const warnings = JSON.parse(result.output)[0].warnings;
+	const warnings = result.results[0].warnings;
 
 	expect(warnings.some((w) => w.text === 'Expected indentation of 0 spaces (indentation)')).toBe(
-		true,
+		false,
 	);
 	expect(warnings.some((w) => w.text === 'Expected "COLOR" to be "color" (property-case)')).toBe(
-		false,
+		true,
 	);
+	expect(result.output).toBe('\na {\nCOLOR: red;\n}');
```

Comment:
[suggest] What about using `stripIndent` for readability?

At L292:

```diff
-	const code = `
+	const code = stripIndent`
		a {
			COLOR: red;
		}`;
```

```suggestion
	expect(result.output).toBe(stripIndent`
		a {
		COLOR: red;
		}`);
```

Reply:
Applied your suggestion slightly modified.
- I have used `stripIndents` instead of `stripIndent` because I want to remove the indentation of all lines
- I have applied `stripIndent` to the source code, because `stripIndents` and `stripIndent` remove an empty line so that the first linebreak of the source-code caused an expect mismatch with the `stripIndents` version it is compared with, because that line was removed entirely.